packages/styled/utils/siteTheme.js

```diff
@@ -0,0 +1,7 @@
+import get from 'lodash/fp/get';
+
+const siteTheme = (valuePath) => (props) => (
+  get(`theme.${valuePath}`, props)
+);
```

Comment:
Can we add support for `lodash/get`'s [`defaultValue` parameter](https://lodash.com/docs/4.17.15#get)?

Reply:
mm, ok, `defaultValue` doesn't exist in the `fp` version of it, but we also don't really need to use the `fp` version.

Reply:
What's fp?